OJP/OJP_JourneySupport.xsd

```diff
@@ -664,14 +657,29 @@
 			</xs:element>
 			<xs:element name="NoRamp" type="xs:boolean" default="false" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>The user is not able to use an ramp.</xs:documentation>
+					<xs:documentation>The user is not able to use a ramp.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="NoSight" type="xs:boolean" default="false" minOccurs="0">
 				<xs:annotation>
 					<xs:documentation>The user is not able to see.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="NoTravelator" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The user is not able to use a travelator.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="LevelEntrance" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The user needs vehicles with level entrance between platform and vehicle.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="LevelEntranceOrBoardingAid" type="xs:boolean" default="false" minOccurs="0">
```

Comment:
Do we need this distinction between LevelEntrance and  LevelEntranceOrBoardingAid? I would understand LevelEntrance  in this way. So maybe we just add the extended annotation from LevelEntranceOrBoardingAid to LevelEntrance?

Reply:
@herlitze Jona had introduced (see #293) the following argument for having both conditions: "In my experience it makes a difference if there is a LevelEntrance or a BoardingAid. A LevelEntrance allows a self-reliant usage of the vehicle while a BoardingAid may involve other people (personnel) to arrange the boarding. Such an assisted boarding is not regarded as a a self-reliant usage of the vehicle."